css/lodesiempre.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@700&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@700&family=Nunito&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+<div class="column-container">
+  <img src="onboarding-assets/img/titulo.png" style="width:502px; margin-top: 15px;">
```

Comment:
inline css is a bad word. please define a specific class for this (.header-logo maybe) and define it in the stylesheet